lib/gRpcRouter.js

```diff
@@ -97,7 +97,13 @@ class GRpcRouter extends Router {
       });
     }
 
-    await this.routeRoute(req, res, chain);
+    const requestLocals = getNamespace('request-locals');
```

Comment:
instead of forcing the grpc plugin with a cls-hooked package and traceId handling in request-local, consider using a separate middleware that wraps `executeRequest` instead